front/logger/withlogging.ts

```diff
@@ -9,18 +9,18 @@ export const statsDClient = new StatsD();
 
 export const withLogging = (handler: any) => {
   return async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
-    let now = new Date();
+    const now = new Date();
     try {
       await handler(req, res);
     } catch (err) {
-      let elapsed = new Date().getTime() - now.getTime();
+      const elapsed = new Date().getTime() - now.getTime();
       logger.error(
         {
           method: req.method,
           url: req.url,
           duration: `${elapsed} ms`,
           error: err,
-          // @ts-expect-error
+          // @ts-expect-error not sure how to do taht properly and make TS happy.
```

Comment:
let me know if you have an opinion or solution for that.
typescript says the following:
```
Property 'stack' does not exist on type '{}' ts(2339)
```